core-tests/src/test/java/org/mule/runtime/core/internal/processor/strategy/ReactorProcessingStrategyTestCase.java

```diff
@@ -200,7 +200,7 @@ public void concurrentStream() throws Exception {
   @Description("If CPU LITE pool is busy OVERLOAD error is thrown")
   public void cpuLightRejectedExecution() throws Exception {
     flow = flowBuilder.get().processors(blockingProcessor)
-        .processingStrategyFactory((context, prefix) -> new ReactorProcessingStrategy(() -> new RejectingScheduler(blocking)))
+        .processingStrategyFactory((context, prefix) -> new ReactorProcessingStrategy(() -> new RejectingScheduler(cpuLight)))
```

Comment:
why this change?

Reply:
This was wrong before, change is unrelated to updgrade.